apps/src/componentLibrary/checkboxDropdown/CheckboxDropdown.story.tsx

```diff
@@ -0,0 +1,280 @@
+import React, {useState, useCallback} from 'react';
+import CheckboxDropdown, {CheckboxDropdownProps} from './index';
+import {Meta, Story} from '@storybook/react';
+
+export default {
+  title: 'DesignSystem/Checkbox Dropdown Component',
+  component: CheckboxDropdown,
+} as Meta;
+
+//
+// TEMPLATE
+//
+// This is needed to fix children type error (passing string instead of React.ReactNode type)
+// eslint-disable-next-line
+const SingleTemplate: Story<CheckboxDropdownProps> = args => {
+  const [selectedValues, setValues] = useState(
+    (args.checkedOptions = [] as string[])
+  );
+  const onChange = useCallback(
+    (e: React.ChangeEvent<HTMLInputElement>) => {
+      // e.stopPropagation();
+      // e.preventDefault();
+      console.log(e);
```

Comment:
Intended logging?

Reply:
Those console log were added for develop,ent consistency, but I agree it's better to remove them from public version. My eyes are bluring already with this component :D

Reply:
haha I can see why, a ton of work here!